com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -626,23 +626,15 @@ private bool ApplyTransformToNetworkStateWithInfo(ref NetworkTransformState netw
                 isPositionDirty = true;
             }
 
-            if (SyncRotAngleX && (Mathf.Abs(Mathf.DeltaAngle(networkState.RotAngleX, rotAngles.x)) >= RotAngleThreshold || networkState.IsTeleportingNextFrame))
+            if (((SyncRotAngleX || SyncRotAngleY || SyncRotAngleZ) && (Mathf.Abs(Mathf.DeltaAngle(networkState.RotAngleX, rotAngles.x)) >= RotAngleThreshold || networkState.IsTeleportingNextFrame)) ||
```

Comment:
This change breaks the original core functionality and would require all of the integration tests to be re-written.  This issue will be resolved in the next update to NetworkTransform where we are using full Quaternion synchronization.

Reply:
@NoelStephensUnity Do we need the network transform changes or do they make this PR redundant?

Reply:
@samlucas-unity 
This PR should be closed.  
#2388 resolves this issue.
_(The editor side updates I added late last week and now I just need to do a final pass update with some of the NetworkTransform integration tests)_